pkg/common/cns-lib/node/nodes.go

```diff
@@ -147,7 +147,7 @@ func (nodes *Nodes) GetNodeNameByUUID(ctx context.Context, nodeUUID string) (
 // This is called by ControllerPublishVolume and ControllerUnpublishVolume
 // to perform attach and detach operations.
 func (nodes *Nodes) GetNodeByUuid(ctx context.Context, nodeUuid string) (*cnsvsphere.VirtualMachine, error) {
-	return nodes.cnsNodeManager.GetNode(ctx, nodeUuid, nil)
+	return nodes.cnsNodeManager.GetNodeVM(ctx, nodeUuid)
```

Comment:
why don't we use GetNodeVM instead of wrapping it in GetNodeByUuid?
if we have removed fss for using node VM UUID, then we should remove this wrapper method.

Reply:
Sure, done!

Reply:
I tried removing the wrapper, but the nodeVM cache is part of the NodeManager and cannot be accessed from outside. I'm leaving the wrapper method as it is.

Reply:
and `GetNodeByUuid` is only used in attach and detach calls?

Reply:
yes